src/components/FileTree.tsx

```diff
@@ -393,14 +393,14 @@ export const FileTreeMenu = () => {
   function createFile() {
     send({
       type: 'Create file',
-      data: { name: '', makeDir: false },
+      data: { name: '', makeDir: false, shouldSetToRename: true },
```

Comment:
I noticed that the process of creating the new file for a sample set it to "renaming mode", as did the text-to-cad process. This makes that step opt-in. This will change when we refactor the file tree to work more like VS Code.